quattor/types/system.pan

```diff
@@ -28,6 +35,105 @@ type structure_oldname = {
     "date" : string
 };
 
+type structure_archetype = {
```

Comment:
can we move these in a `quattor/types/aquilon` template and include it here? i don't like this one growing again